presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -5301,6 +5301,86 @@ public void testUnsupportedCsvTable()
                 "\\QHive CSV storage format only supports VARCHAR (unbounded). Unsupported columns: i integer, bound varchar(10)\\E");
     }
 
+    @Test
```

Comment:
Moving the Tests from AbstractTestDistributedQueries to TestHiveIntegrationSmokeTest.

Figured that AbstractTestDistributedQueries is being invoked by every connector/catalog, and catalogs other than "hive" don't support partitioned MV. This leads to test failures or previous build failure.

This line, https://github.com/prestodb/presto/blob/9f89256256c4bb577b8a9e78c24047c49da93275/presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java#L76, indicates Hive specific tests should go here.

https://github.com/prestodb/presto/blob/9f89256256c4bb577b8a9e78c24047c49da93275/presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java#L76